x/concentrated-liquidity/incentives_test.go

```diff
@@ -2783,7 +2791,7 @@ func (s *KeeperTestSuite) TestQueryAndClaimAllIncentives() {
 			growthOutside:    uptimeHelper.twoHundredTokensMultiDenom,
 			numShares:        sdk.OneDec(),
 
-			expectedError: types.NegativeDurationError{Duration: time.Hour * 504 * -1},
+			expectedError: types.NegativeDurationError{Duration: time.Hour * 336 * -1},
```

Comment:
Note to reviewer: the difference here is equivalent to the amount by which the longest uptime increased over our previous version, which is 1 week